roles/deploy/tasks/main.yml

```diff
@@ -146,10 +151,9 @@
 #-------------------------
 # Move new code into place
 
-- name: move build to current
-  command: mv {{ build_path }} {{ current_path }} removes={{ build_path }}
-  become: yes
-  become_user: "{{ unicorn_user }}"
+- name: copy build to current
+  command: cp -TLr {{ build_path }} {{ current_path }}
```

Comment:
so basically, our `releases/current/` is not a symlink but a copy

Reply:
Yeah, I was getting weird results when looking at the files copied into `/rollback`. I think it was because a lot of the files in `/build` are symlinks. Strangely, about 50% of the time the files were not all there. Forcing the copy to dereference symlinked files and create real copies fixed the issue and made the outcomes repeatable, with all the files intact each time.

Reply:
If you have any insights on this one, they would be appreciated!

Reply:
@enricostano you might have a better idea as you played with [ansistrano/deploy](https://github.com/ansistrano/deploy) in the past.

Reply:
@Matt-Yorkley worth checking https://github.com/ansistrano/rollback